src/notation/qml/MuseScore/NotationScene/SelectionFilterPanel.qml

```diff
@@ -30,194 +30,272 @@ import Muse.Ui 1.0
 
 import "internal"
 
-StyledFlickable {
+Item {
     id: root
 
     property NavigationSection navigationSection: null
     property int navigationOrderStart: 0
 
-    contentWidth: root.width
-    contentHeight: contentColumn.implicitHeight
+    readonly property bool isAllSelected: voicesModel.isAllSelected && notesInChordModel.isAllSelected && elementsModel.isAllSelected
+    readonly property bool isNoneSelected: voicesModel.isNoneSelected && notesInChordModel.isNoneSelected && elementsModel.isNoneSelected
 
-    Component.onCompleted: {
-        voicesModel.load()
-        notesInChordModel.load()
-        elementsModel.load()
-    }
-
-    Column {
-        id: contentColumn
-
-        readonly property real buttonHeight: 24
+    StyledFlickable {
+        id: flickable
 
         anchors {
-            fill: parent
-            leftMargin: 12
-            rightMargin: 12
+            top: root.top
+            left: root.left
+            bottom: clearAllSelectAllRow.top
         }
 
-        spacing: 10
+        width: root.width
+        height: root.height
```

Comment:
Doesn't this value for `height` conflict with the top and bottom anchor? Let's perhaps use exclusively anchors here.